inst/rstudio/templates/tech_report/project/techreport_resources/template.tex

```diff
@@ -4,32 +4,32 @@
 \usepackage{ifxetex,ifluatex}
 \usepackage{fixltx2e} % provides \textsubscript
 \ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
-\usepackage[T1]{fontenc}
-\usepackage[utf8]{inputenc}
+  \usepackage[T1]{fontenc}
```

Comment:
I don't think these formatting changes are necessary. You shouldn't need to make any changes to the template.tex file in the IE Usage report repo.